modules/tinymce/src/plugins/media/test/ts/atomic/UpdateHtmlTest.ts

```diff
@@ -0,0 +1,37 @@
+import * as UpdateHtml from '../../../main/ts/core/UpdateHtml';
```

Comment:
Import path is incorrect here. It should be `tinymce/plugins/media/core/UpdateHtml`

Reply:
When do we want relative paths and when do we want the `tinymce/...` types? I'm not feeling quite clear on it.

Reply:
The rule is if it's in the same directory (eg `main` or `test) it should be relative. If it's outside of that then it should be an alias.

Reply:
Makes sense. I'll keep to it.